pages/summary.js

```diff
@@ -2,28 +2,80 @@ import PropTypes from "prop-types";
 import { API_URL } from "../lib/constants";
 import { getQueryStringFromServerQuery } from "../lib/query-utils";
 import Breadcrumbs from "../components/breadcrumbs";
-import JsonDisplay from "../components/json-display";
+import { DataPanel, DataAreaTitle } from "../components/data-area";
+import { ButtonLink } from "../components/form-elements";
 import PagePreamble from "../components/page-preamble";
+import RegulomeVersionTag from "../components/regulome-version-tag";
+import SummaryTable from "../components/summary-table";
+import Notifications from "../components/notifications";
 
-export default function Summary({ data }) {
+export default function Summary({ data, queryString }) {
+  const total = data.total || 0;
+  const variants = [];
+  const assembly = data.assembly;
+  if (total > 1) {
+    for (let i = 0; i < data.variants.length; i++) {
+      const variant = [];
```

Comment:
I think this should be `const variant = {}`. Initializing it to an array is legal, but it means the result will include an empty array that has additional properties like `chrom_location`, etc.

Reply:
You are right